developer_tests/utilities/parse_args_test.f90

```diff
@@ -1,90 +1,198 @@
 ! DART software - Copyright UCAR. This open source software is provided
 ! by UCAR, "as is", without charge, subject to all terms of use at
 ! http://www.image.ucar.edu/DAReS/DART/DART_download
-!
-
-! this test reads stdin.  to automate it, run it with:
-!  cat bob | ./parse_args_test
-! or
-!  ./parse_args_test < bob
-!
-! where the file bob contains one line per test with arguments to be parsed.
 
 program parse_args_test
 
-use utilities_mod,     only : initialize_utilities, finalize_utilities, &
-                              register_module, error_handler, E_ERR, E_MSG
+use utilities_mod,     only : initialize_utilities, finalize_utilities
 use parse_args_mod,    only : get_args_from_string, get_name_val_pairs_from_string
 
-implicit none
+use test  ! fortran-testanything
 
-! version controlled file description for error handling, do not edit
-character(len=*), parameter :: source   = "parse_args_test.f90"
-character(len=*), parameter :: revision = ""
-character(len=*), parameter :: revdate  = ""
+implicit none
 
-integer :: iunit = 5
-integer :: ierr, nargs, i
+integer, parameter :: MAXVALS = 100
+integer :: nargs
 character(len=512) :: nextline
-character(len=64) :: args(100)
-character(len=64) :: names(100)
-character(len=64) :: vals(100)
+character(len=64) :: args(MAXVALS)
+character(len=64) :: names(MAXVALS)
+character(len=64) :: vals(MAXVALS)
 logical :: continue_line
 
+character(len=20) :: testnum
+
+! the ascii code for the backslash character is 92.
+character(len=1), parameter :: BACKSLASH = ACHAR(92)
+
 !----------------------------------------------------------------------
 
 ! main code here
  
 ! initialize the dart libs
 call initialize_utilities('parse_args_test')
```

Comment:
```suggestion
call initialize_utilities('parse_args_test', standalone_program=.true.)
```

I don't think this test program needs to write namelists, logs so if possible it would be good to only output the test output to std out that way you can pipe it to something like tapview:

```
./parse_args_test | ./tapview
.........................................................
57 tests, 0 failures.
```

Reply:
fixed